client/web/src/components/widgets/inputs/SwatchPairInput.vue

```diff
@@ -28,6 +28,7 @@
 		position: relative;
 
 		button {
+			--swatch-color: white;
```

Comment:
If you don't mind switching this to `#ffffff`, it would be preferred over the name `white`.